Added context to the root folder

saai-syvendra opened the pull request:
fixed the issue by adding context

saai-syvendra committed:
Added context to the root folder

fixed the issue by adding context

yumyum-web commented on docker-compose.yaml:
```diff
@@ -1,168 +1,177 @@
+        container_name: financial-security-system-credit-risk-scoring
+        environment:
+            KAFKA_BROKERS: kafka:29092
+            DATABASE_URL: postgresql+psycopg2://postgres:password@postgres:5432/financial_security_system
```
Docker compose only runs on dev.

yumyum-web committed:
Refine .dockerignore patterns for better matching

Update ignore patterns to include leading wildcards for all relevant
files and directories. This ensures that Docker ignores files across
all subdirectories.

yumyum-web commented:
The reason we ended up including venv (along with a host of other files) in every service image was all thanks to a rather lenient `.dockerignore`.
